dev-docs/bidders/stellormedia.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: bidder
+title: StellorMedia
+description: Stellor Media Bidder Adapter
+biddercode: stellormedia
+aliasCode: adtelligent
+media_types: video,banner
+tcfeu_supported: true
```

Comment:
If you claim tcf support, you must add a `gvl_id` here. If the gvl_id belongs to `adtelligent` you should add this, too. See example 

https://github.com/prebid/prebid.github.io/blob/59260bde6883ff0bad1b9f934d822afc2e871d05/dev-docs/bidders/adblender.md?plain=1#L9